packages/now-client/src/create-deployment.ts

```diff
@@ -1,205 +1,170 @@
-import { readdir as readRootFolder, lstatSync } from 'fs-extra';
-
+import { omit } from 'rambda';
+import { upload } from './upload';
 import readdir from 'recursive-readdir';
+import { DeploymentError } from './errors';
 import { relative, join, isAbsolute } from 'path';
 import hashes, { mapToObject } from './utils/hashes';
-import { upload } from './upload';
+import { readdir as readRootFolder, lstatSync } from 'fs-extra';
 import { getNowIgnore, createDebug, parseNowJSON } from './utils';
-import { DeploymentError } from './errors';
 import {
   NowConfig,
   NowClientOptions,
   DeploymentOptions,
   DeploymentEventType,
 } from './types';
 
-export default function buildCreateDeployment(version: number) {
```

Comment:
Should we detect v1 and yield an `error`? 🤔

Reply:
Nope, everything 1.0 related is gone.